scripts/lib/CIME/XML/batch.py

```diff
@@ -93,10 +93,6 @@ def get_value(self, name, attribute=None, resolved=True, subgroup=None):
         if node is not None:
             value = self.text(node)
 
-        if value is None:
```

Comment:
This pattern was broken. The GenericXML.get_value is a placeholder that just always returns None.